web-module/src/main/webapp/resources/structEditJS/autocomplete.js

```diff
@@ -0,0 +1,120 @@
+(function(window, undefined) {
```

Comment:
1. please add a CELDEV issue for "refactoring autocomplete.js as js-class" and mention it on the first line as TODO-comment
2. add the LGPL opensource license comment